packages/paste-design-tokens/tokens/global/box-shadow.yml

```diff
@@ -158,6 +158,11 @@ props:
     value: "0px -1px {!offset-0} {!palette-blue-90}"
     comment: Top shadow border for selected hover status of horizontal tabs
 
+  # End borders
+  shadow-border-end-inverse-strong:
```

Comment:
question: is there a reason we'd want to do this, instead of explicit "-right" and "-bottom" shadow tokens?

Reply:
In the context of borders and tokens-to-css properties, I feel like `-right` would be better